crates/bindings-csharp/Codegen.Tests/fixtures/server/snapshots/Module#MultiTableRow.verified.cs

```diff
@@ -92,18 +94,7 @@ SpacetimeDB.BSATN.ITypeRegistrar registrar
                     Indexes: [],
                     Constraints:
                     [
-                        new(
```

Comment:
@lcodes This looks like a notable regression. Shouldn't `Foo` still be autoinc in MultiTable2?